### ISSUE TITLE
No Device found in nodered

### ISSUE THREAD

**Issue by SpeedyRS2** [hardillb/node-red-alexa-home-skill-web#57]
I have a problem with the alexa skill.

I added a device under https://alexa-node-red.bm.hardill.me.uk/devices in my account and installed the alexa skill in the alexa app.
![image](https://user-images.githubusercontent.com/4572175/37706541-273a759e-2d00-11e8-9501-aeeac9698cc0.png)

After searching for devices in the alexa App, the 2 new devices are found.
![image](https://user-images.githubusercontent.com/4572175/37706566-408a681a-2d00-11e8-8c59-8adf34f0b158.png)

but the nodered plugin doesn't find any device, even after refresh. The node is connected, but the device list stays empty.
![image](https://user-images.githubusercontent.com/4572175/37705203-8744fd56-2cfb-11e8-9302-8f58a531a11c.png)

Do you have any suggestions for me?

Thanks and best regards,
Andreas

**Comment by hardillb** [comment]
I'll try and have a look tonight

**Comment by hardillb** [comment]
Can you check the node-red logs to see if there is any output when you hit the refresh button

**Comment by hardillb** [comment]
Also what version of Node-RED and NodeJS are you using

**Comment by SpeedyRS2** [comment]
Good morning and thank you for your repley.

22 Mar 08:07:05 - [info] Node-RED version: v0.18.3
22 Mar 08:07:05 - [info] Node.js  version: v6.13.0
22 Mar 08:07:05 - [info] Linux 4.14.18-v7+ arm LE

The Log, even in trace mode dosn't contain any entrys. There is no output from the plugin.

**Comment by SpeedyRS2** [comment]
I extended the log at line 327 a little bit:
err: null
id: 7d5d9396.22e9bc
devicesURL: https://alexa-node-red.bm.hardill.me.uk/api/v1/devices
username: SpeedyRS2
password: xxxxxxxxx
res: [object Object]
body: Unauthorized

ok, i found the problem. I used a password with special characters like ":" and so on. This was the problem. I registred a new user, with another password (without  special characters) and the device was found from the plugin.

Thank you very much for your help.
